homeassistant/components/wemo/light.py

```diff
@@ -150,45 +152,70 @@ def available(self):
 
     def turn_on(self, **kwargs):
         """Turn the light on."""
-        transitiontime = int(kwargs.get(ATTR_TRANSITION, 0))
-
+        brightness = None
+        xy_color = None
+        color_temp = None
+        transition_time = int(kwargs.get(ATTR_TRANSITION, 0))
         hs_color = kwargs.get(ATTR_HS_COLOR)
 
         if hs_color is not None:
             xy_color = color_util.color_hs_to_xy(*hs_color)
-            self.wemo.set_color(xy_color, transition=transitiontime)
 
         if ATTR_COLOR_TEMP in kwargs:
-            colortemp = kwargs[ATTR_COLOR_TEMP]
-            self.wemo.set_temperature(mireds=colortemp, transition=transitiontime)
+            color_temp = int(kwargs.get(ATTR_COLOR_TEMP))
```

Comment:
Don't use dict.get when we know the key is in the dict. Use `dict[key]`.

The color temperature service data value is already validated as an integer.

Reply:
If I switch this back, it'll make the pylint check fail. Is that ok?

Reply:
Why does it fail?

Reply:
The pylint validation error told me to switch this to a dict.get:

************* Module homeassistant.components.wemo.light
homeassistant/components/wemo/light.py:164:8: R1715: Consider using dict.get for getting values from a dict if a key is present or a default if not (consider-using-get)

Reply:
Even though the language was "consider", this still failed the check.

Reply:
That's from here: https://dev.azure.com/home-assistant/Core/_build/results?buildId=43633&view=logs&j=8b66295a-05e1-52cf-5640-7a4f4e1975e2&t=6c4465fe-285a-5650-787c-2e3dbe1da192 in case you want to take a look at the log.

Reply:
Pylint wants us to remove the first check and just use `dict.get`.